packages/flutter/lib/src/cupertino/sliding_segmented_control.dart

```diff
@@ -52,6 +52,10 @@ const Radius _kSeparatorRadius = Radius.circular(_kSeparatorWidth / 2);
 // amount of time.
 const double _kMinThumbScale = 0.95;
 
+// The maximum scale factor of the thumb, when being pressed on for a sufficient
+// amount of time.
```

Comment:
What is the sufficient amount of time this is referring to?

Reply:
Its same as `_kMinThumbScale` i.e. duration of a press that triggers the thumb scaling animation.

Reply:
It seems like this value is only used for the momentary variant, and the name doesn't make much sense otherwise, and definitely doesn't have a parallel meaning with `_kMinThumbScale`. Therefore I suggest:
```suggestion
// The peak scale factor of the thumb during the pressing animation of a momentary variant.
const double _kMaxThumbScaleForMomentary = 1.05;
```